debian/trusty/foreman/control

```diff
@@ -3,10 +3,10 @@ Maintainer: Greg Sutcliffe <greg.sutcliffe@gmail.com>
 Section: web
 Priority: extra
 Standards-Version: 3.9.1
-Build-Depends: debhelper (>= 7), cdbs, bundler (>= 1.3.5-2ubuntu2), ruby2.0, ruby2.0-dev, git | git-core,
+Build-Depends: debhelper (>= 9.20160709), bundler (>= 1.3.5-2ubuntu2), ruby2.0, ruby2.0-dev, git | git-core,
```

Comment:
Trusty only has `9.20131227ubuntu1`, and I have no idea about trusty backports

Reply:
Foreman 1.15 is the last version to support trusty, so this can be ommited alltogether.